tests/run-macros/reflect-inline/assert_1.scala

```diff
@@ -5,7 +5,7 @@ object api {
     ${ stripImpl(x) }
 
   private def stripImpl(x: String)(given qctx: QuoteContext): Expr[String] =
-    Expr(x.stripMargin)
+    Expr(augmentString(x).stripMargin)
```

Comment:
For some reason the `inline` argument checks changed behaviour. Before, when it tried the extension method `stripMargin` above it failed when the argument was not a constant and then found the `augmentString` conversion. Now it types it as the the extension method with an `inline` String but does then does not fail due to the non constantness of the argument. This is also the source of the issue in `scalatest`.

Reply:
The new resolution `stripMargin` seems better, but the `inline` parameter should have failed.

Reply:
This is only the case for the prefix in an extension method.